tools/buildtools/ambertools/antechamber.xml

```diff
@@ -0,0 +1,150 @@
+<tool id="antechamber" name="AnteChamber" version="@VERSION@">
+  <description>Amber's molecular input file processor</description>
+  <macros>
+    <import>macros.xml</import>
+  </macros>
+  <expand macro="requirements"></expand>
+  <command detect_errors="exit_code">
+    <![CDATA[
+        antechamber -i '$input1' -o '$output1' -fi '$input_format.selected_format' -fo '$selected_output_format' -c '$allparams.c' -pf '$extraparams.pf'
+        #if '$extraparams.usenc'
+        -nc '$allparams.nc'
+        #end if
+        -at '$allparams.at' -j '$allparams.j' -rn '$allparams.resname' -m '$allparams.m'
+        ]]>
+  </command>
+  <inputs>
+    <conditional name="input_format">
+      <param name="selected_format" type="select" label="Select a file format:" help="In PDB or mol2 format">
+        <option value="pdb" selected="true">pdb</option>
+        <option value="mol2">mol2</option>
+      </param>
+      <when value="pdb">
+        <param type="data" name="input1" format="pdb"/>
+      </when>
+      <when value="mol2">
+        <param type="data" name="input1" format="mol2"/>
+      </when>
+    </conditional>
+    <param name="selected_output_format" type="select" label="Select an output file format (mol2 recommended!):">
+      <option value="pdb">pdb</option>
+      <option value="mol2" selected="true">mol2</option>
+    </param>
+    <section name="allparams" title="General Parameters" expanded="true">
+      <param name="nc" type="integer" value="0" label="Net charge. Default is 0" min="-50" max="50" help="Specify the net charge for the molecule. (-nc)"/>
```

Comment:
It's a minor point but I'd be tempted to move the 'Default is 0' to the help section. And the same for the multiplicity and residue name below

Reply:
Move the text about default options, or do you mean move all these params to another section?

Reply:
No just the text

Reply:
made it a little less bulky, see in the next commit coming shortly.

Reply:
Sorry - for clarification, I don't mean the help section, but the help attribute for the parameter. E.g.  `help="Specify the net charge for the molecule. (-nc) Default is 0."/>`